case-lib/lib.sh

```diff
@@ -357,6 +384,44 @@ func_mtrace_collect()
     sudo "${mtraceCmd[@]}" >& "$clogfile" &
 }
 
+func_lib_log_post_process()
+{
+    # nothing to do unless dictionary backends of Zephyr
+    # are used
+    if ! is_ipc4 && ! is_firmware_file_zephyr; then
+        return 0
```

Comment:
You can avoid all the negations and the `if/then/else` using the common "assert || return" style

```shell
 is_ipc4 || is_firmware_file_zephyr || return 0
```

BTW this condition looks weird, could use a comment.

Reply:
Fixed.